doc/release-notes.md

```diff
@@ -36,6 +36,13 @@ No attempt is made to prevent installing or running the software on Windows XP,
 you can still do so at your own risk, but do not expect it to work: do not
 report issues about Windows XP to the issue tracker.
 
+From 0.13.1 onwards OS X 10.7 is no longer supported. Issues with the libc++
```

Comment:
So maybe instead this should read something like: "0.13.0 was intended to work on 10.7+, but severe issues with the libc++ version on 10.7.x keep it from running reliably. 0.13.1 now requires 10.8+, and will communicate that to 10.7 users, rather than crashing unexpectedly".